packages/mdc-line-ripple/README.md

```diff
@@ -42,15 +42,15 @@ CSS Class | Description
 --- | ---
 `mdc-line-ripple` | Mandatory
 `mdc-line-ripple--active` | Styles the line ripple as an active line ripple
-`mdc-line-ripple--deactivating` | Styles the line ripple as a deactivating line ripple. 
+`mdc-line-ripple--deactivating` | Styles the line ripple as a deactivating line ripple
 
 ### Sass Mixins
```

Comment:
I think this is unnecessary

Reply:
Hm. Actually it looks like there _is_ a public mixin but it was never documented. Guess I'll add that in here.